packages/web/graphql/userConfiguration/updateUserConfiguration.graphql

```diff
@@ -0,0 +1,5 @@
+mutation updateUserConfiguration($digestEmailConfig: DigestEmailConfiguration) {
```

Comment:
@Lanny just FYI I found that you actually can reference a custom type within the args in these definitions!